wwr/about.html

```diff
@@ -0,0 +1,86 @@
+    </nav>
+  </header>
+  <main>
+    <div class="hero">
+      <h1>Rafting adventure</h1>
+      <article>
+        <p class="mission">"At Rafting Adventures, our purpose is to offer thrilling, safe, and eco-friendly rafting
+          adventures that connect people with nature, promote teamwork, and inspire environmental stewardship."</p>
+        <img class="little-hero" src="images/little-hero.jpg" alt="Happy customers" width="100" height="100">
+      </article>
+    </div>
+    <section class="history">
+      <h2>History</h2>
+      <div class="history-content">
+      <p>Our company was founded by a group of lifelong friends with a passion for rafting and a deep
+        love for nature. Committed to sustainability, they use environmentally responsible practices and support local
+        conservation efforts. Their mission is to create</p>
+      <img class="history-content" id="his-cen" src="images/img7.png">
+      <p class="history-content">unforgettable experiences while fostering a sense of stewardship for the natural world.
+        Inspired by their experiences growing up near rivers, they created a company focused on providing thrilling,
+        eco-friendly adventures that connect people with the outdoors.</p>
+      </div>
+    </section>
+    <section>
+      <h2 id="adventures">Adventure Awaits You!</h2>
+      <div class="figure-container">
+      <figure>
+        <img class="adventure" src="images/img1.jfif" alt="Image of people living the adventure 1">
+        <figcaption>The greatest adventure of all times</figcaption>
+      </figure>
+      <figure>
+        <img class="adventure" src="images/img2.webp" alt="Image of people living the adventure 2">
+        <figcaption>Our friends love our programs</figcaption>
+      </figure>
+      <figure>
+        <img class="adventure" src="images/img3.jfif" alt="Image of people living the adventure 3">
+        <figcaption>Discover our rafting adventure</figcaption>
+      </figure>
+      <figure>
+        <img class="adventure" src="images/img4.jfif" alt="Image of people living the adventure 4">
+        <figcaption>Our destinies will astonish you</figcaption>
+      </figure>
+      <figure>
+        <img class="adventure" src="images/img5.jfif" alt="Image of people living the adventure 5">
+        <figcaption>Come with your loved ones</figcaption>
+      </figure>
+      <figure>
+        <img class="adventure" src="images/img6.jfif" alt="Image of people living the adventure 6">
+        <figcaption> Enjoy the best experience ever!</figcaption>
+      </figure>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <p id="pfoot">© 2024 │ Rafting Adventure │ Josue Leyva</p>
+    <div class="sociallinks">
+      <i id="facebook" class="fa fa-facebook-official" style="font-size: 36px"></i>
+      <i id="instagram" class="fa fa-instagram" style="font-size: 36px"></i>
+      <i id="youtube" class="fa fa-youtube-play" style="font-size: 36px"></i>
```

Comment:
This kind of styling is not allowed in this course, please move to the `rafiting.css`

![image](https://github.com/user-attachments/assets/0c925982-1758-43fa-905c-2b3840480281)